File: internal/service/testdata/matrix/basic.cs
using System;

namespace Demo {
  interface INamed {
    string Name { get; }
  }

  struct Point {
    public int X;
  }

  enum Color { Red, Blue }

  class Greeter {
    private int count = 0;
    public string Label { get; set; }

    public Greeter() {}

    public int Helper() {
      int local = 1;
      return local;
    }

    public void Main() {
      Helper();
      Helper();
    }
  }
}

File: internal/service/testdata/matrix/basic.rs
use std::fmt;

mod utils {}

struct Point {
    x: i32,
}

enum Color {
    Red,
    Blue,
}

trait Drawable {
    fn draw(&self);
}

impl Drawable for Point {
    fn draw(&self) {}
}

type ID = i32;

fn helper() -> i32 {
    let x = 1;
    x
}

fn main() {
    let _ = helper();
    helper();
}
